model/api/src/main/java/org/opennms/oce/model/api/ModelObject.java

```diff
@@ -36,12 +36,36 @@
 
     Set<ModelObject> getChildren();
 
+    Set<ModelObject> getPeers();
```

Comment:
Since we're using relations for this Interface, shouldn't this be getSiblings()?

Reply:
That would make sense....